cmake/qtifw/install_win_copydll.qs

```diff
@@ -50,9 +51,9 @@ function Component()
 
       // Store ocx to be registered
       var dllsToReg = [];
-      for (var i = 0; i < systemArray.length; i++) {
-        var sourceFile = `${tempDir}\\${systemArray[i]}`;
-        var targetFile = systemTargetDir + systemArray[i];
+      systemArray.forEach(dll => {
+        var sourceFile = `${tempDir}\\${dll}`;
+        var targetFile = `${systemTargetDir}\\${dll}`;
```

Comment:
The rest is just clean up in the scripts to make @Myoldmopar happy with the use of forEach loops and co